core/modules/random_drop.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BESS_MODULES_RANDOM_DROP_H_
+#define BESS_MODULES_RANDOM_DROP_H_
+
+#include "../module.h"
+#include "../module_msg.pb.h"
+#include "../utils/random.h"
+
+class RandomDrop final : public Module {
+ public:
+  pb_error_t Init(const bess::pb::RandomDropArg &arg);
+  void ProcessBatch(bess::PacketBatch *batch) override;
+
+ private:
+  Random rng_;
+  uint threshold_;
+  const uint range_ = 1000000;  // for granularity
```

Comment:
Could you define this in the location / style for constants per the Google C++ style guide?